hooks/pre_gen_project.py

```diff
@@ -6,27 +6,27 @@
 
 TODO: ? restrict Cookiecutter Django project initialization to Python 3.x environments only
 """
+from __future__ import print_function
+
+import sys
+
+TERMINATOR = "\x1b[0m"
+WARNING = "\x1b[1;33m [WARNING]: "
+INFO = "\x1b[1;33m [INFO]: "
+HINT = "\x1b[3;33m"
+SUCCESS = "\x1b[1;32m [SUCCESS]: "
```

Comment:
Could we share them with `post_gen_project.py`?

Reply:
Nope, tried that already: `cookiecutter` refuses to recognize `hooks` as a package, with or without `__init__.py` inside which prevented from refactoring these declarations into a distinct module. Maybe @pydanny could suggest something?